compute/cker/include/cker/operation/DepthwiseConv.h

```diff
@@ -180,6 +182,47 @@ inline void DepthwiseConv(const DepthwiseConvParams &params, const Shape &input_
   cpu_backend_threadpool::Execute(tasks.size(), tasks.data(), ruy_context);
 }
 
+template <typename T>
+void DepthwiseConvOp(const DepthwiseConvParams &params, const Shape &input_shape,
+                     const T *input_data, const Shape &filter_shape, const T *filter_data,
+                     const Shape &bias_shape, const T *bias_data, T *padded_filter_data,
+                     bool pad_filter, T *filter_buffers_data, const Shape &output_shape,
+                     T *output_data)
+{
+  if (params.stride_height != params.stride_width)
+    throw std::runtime_error("Not support different length strides");
+
+  if (params.dilation_height_factor != 1 || params.dilation_width_factor != 1)
+    throw std::runtime_error{"Not support dilation other than 1."};
+
+  const int batch = MatchingDim(input_shape, 0, output_shape, 0);
+  const int input_depth = input_shape.Dims(3);
+  const int output_depth = output_shape.Dims(3);
+  const int input_height = input_shape.Dims(1);
+  const int input_width = input_shape.Dims(2);
+  const int filter_height = filter_shape.Dims(1);
+  const int filter_width = filter_shape.Dims(2);
+  const int output_height = output_shape.Dims(1);
+  const int output_width = output_shape.Dims(2);
+  const int stride = params.stride_height;
+  const int depth_multiplier = params.depth_multiplier;
+  const int pad_height = params.padding_values.height;
+  const int pad_width = params.padding_values.width;
+  const T activation_min = params.float_activation_min;
+  const T activation_max = params.float_activation_max;
```

Comment:
I'm not sure if non-float types use `params.float_activation_min` and we have to support those types for DepthwiseConv op. So, I think it's better to change this function from function template to normal function that supports only `float`.